src/plugins/dashboard/public/dashboard_actions/copy_to_dashboard_modal.tsx

```diff
@@ -88,90 +92,70 @@ export function CopyToDashboardModal({
   const descriptionId = 'copyToDashboardDescription';
 
   return (
-    <EuiFocusTrap clickOutsideDisables={true}>
-      <EuiOutsideClickDetector onOutsideClick={closeModal}>
-        <div
-          role="dialog"
-          aria-modal="true"
-          aria-labelledby={titleId}
-          aria-describedby={descriptionId}
-        >
-          <PresentationUtilContext>
```

Comment:
I notice we are no longer wrapping in `<PresentationUtilContext>`. Can we remove it from the props?